lib/index.js

```diff
@@ -96,6 +103,21 @@ const updateAvailFiats = fiats => ({
   payload: fiats
 });
 
+const setFeed = feed => ({
+  type: 'SET_FEED',
+  payload: feed
+});
+
+const updateFiat = fiat => ({
+  type: 'UPDATE_FIAT',
+  payload: fiat
+});
+
+const refreshPrice = {
```

Comment:
getting an error because this isn't a function. Needs to be in the same format as the other action creators.